libraries/cli/CMakeLists.txt

```diff
@@ -19,6 +15,44 @@ target_include_directories(cli PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
 target_link_libraries(cli PUBLIC config)
 
 install(TARGETS cli
-    ARCHIVE  DESTINATION ${CMAKE_INSTALL_LIBDIR}
-    LIBRARY  DESTINATION ${CMAKE_INSTALL_LIBDIR}
-)
\ No newline at end of file
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+)
+
+set(JSONS_DIRECTIORY ${CMAKE_CURRENT_SOURCE_DIR}/include/cli/config_jsons)
+
+file(READ ${JSONS_DIRECTIORY}/mainnet.json MAINNET_CONFIG_JSON)
+file(READ ${JSONS_DIRECTIORY}/testnet.json TESTNET_CONFIG_JSON)
+file(READ ${JSONS_DIRECTIORY}/devnet.json DEVNET_CONFIG_JSON)
+
+# no maps in cmake, so it is organized as two arrays. Order of this and config_files should be the same
+set(configs
+    DEFAULT_CONFIG_JSON
+    MAINNET_CONFIG_JSON
+    TESTNET_CONFIG_JSON
+    DEVNET_CONFIG_JSON
+)
+set(config_files
+    ${JSONS_DIRECTIORY}/default.json
+    ${JSONS_DIRECTIORY}/mainnet.json
+    ${JSONS_DIRECTIORY}/testnet.json
+    ${JSONS_DIRECTIORY}/devnet.json
+)
+
+list(LENGTH configs configs_count)
+# length should be less by 1 for foreach
+math(EXPR configs_count "${configs_count}-1")
+foreach(i RANGE ${configs_count})
+    list(GET configs ${i} config_name)
+    list(GET config_files ${i} config_file)
+    file(READ ${config_file} ${config_name})
+
+    set(config_str ${${config_name}})
+
+    string(JSON result ERROR_VARIABLE json_error TYPE ${config_str} chain_config)
+    if(NOT(${result} STREQUAL "OBJECT"))
+        message(FATAL_ERROR "Can't parse ${config_name}: ${json_error}")
+    endif()
+endforeach()
```

Comment:
is this where we validate json files? I would add some comment here what is this code doing